packages/amplication-server/src/core/git/git.provider.service.ts

```diff
@@ -47,6 +50,13 @@ export class GitProviderService {
     );
   }
 
+  async createGitClient(
+    gitProviderArgs: GitProviderArgs,
+    logger = this.logger
+  ): Promise<GitClientService> {
+    return new GitClientService().create(gitProviderArgs, logger);
```

Comment:
@yuval-hazaz we use this service in git-pull-request service, but only one. Should we do the same there?
https://github.com/amplication/amplication/blob/master/packages/amplication-git-pull-request-service/src/pull-request/pull-request.service.ts#L46

Reply:
we should talk separately about git-pull-request  - how would we fetch the org properties from the DB?